os_stub/spdm_device_secret_lib_sample/raw_data_key.h

```diff
@@ -0,0 +1,442 @@
+/**
+ *  Copyright Notice:
+ *  Copyright 2021-2022 DMTF. All rights reserved.
+ *  License: BSD 3-Clause License. For full text see link: https://github.com/DMTF/libspdm/blob/main/LICENSE.md
+ **/
+
+/*automatic generated by raw_data_key_gen.py*/
+
+#include "spdm_device_secret_lib_internal.h"
+
+#ifndef LIBSPDM_RAW_DATA_KEY

```

Comment:
it should be RAW_DATA_KEY_H, to match the file name.

Reply:
Thanks. I got it. I will remember the rule.